includes/admin/extensions/product-education/class-wpsmtp.php

```diff
@@ -0,0 +1,232 @@
+				<?php esc_html_e( 'WP Mail SMTP allows you to easily set up WordPress to use a trusted provider to reliably send emails, including sales notifications.', 'easy-digital-downloads' ); ?>
+			</p>
+
+			<div class="edd-extension-manager__group">
+				<div class="edd-extension-manager__step">
+					<?php $this->manager->button( $this->get_button_parameters() ); ?>
+				</div>
+
+				<?php
+				if ( $this->is_smtp_activated() ) {
+					return;
+				}
+				?>
+				<div class="edd-extension-manager__step" style="display:none;">
+					<?php $this->manager->link( $this->get_link_parameters() ); ?>
+				</div>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Gets the button parameters.
+	 *
+	 * @return array
+	 */
+	private function get_button_parameters() {
+		$button = array();
+		// If neither the lite nor pro plugin is installed, the button will prompt to install and activate the lite plugin.
+		if ( ! $this->manager->is_plugin_installed( $this->config['lite_plugin'] ) && ! $this->manager->is_plugin_installed( $this->config['pro_plugin'] ) ) {
+			$button['plugin']      = $this->config['lite_download_url'];
+			$button['action']      = 'install';
+			$button['button_text'] = __( 'Install & Activate WP Mail SMTP', 'easy-digital-downloads' );
+		} elseif ( ! $this->is_smtp_activated() ) {
+			// If one of the SMTP plugins is installed, but not activated, the button will prompt to activate it.
+			$button['plugin']      = $this->config['lite_plugin'];
+			$button['action']      = 'activate';
+			$button['button_text'] = __( 'Activate WP Mail SMTP', 'easy-digital-downloads' );
+		} elseif ( ! $this->is_smtp_configured() ) {
+			// If the plugin is active, but not configured, the button will send them to the setup wizard.
+			$button = $this->get_link_parameters();
+		}
+
+		return $button;
+	}
+
+	/**
+	 * Gets the array of parameters for the link to configure WP Mail SMTP.
+	 *
+	 * @since 2.11.x
+	 * @return array
+	 */
+	private function get_link_parameters() {
+		return $this->is_smtp_configured() ?
+		array(
+			'button_text' => __( 'Configure WP Mail SMTP', 'easy-digital-downloads' ),
+			'href'        => admin_url( $this->config['smtp_settings'] ),
+		) :
+		array(
+			'button_text' => __( 'Run the WP Mail SMTP Setup Wizard', 'easy-digital-downloads' ),
+			'href'        => admin_url( $this->config['smtp_wizard'] ),
+		);
+	}
+
+	/**
+	 * Whether WP Mail SMTP plugin configured or not.
+	 *
+	 * @since 2.11.x
+	 *
+	 * @return bool True if some mailer is selected and configured properly.
+	 */
+	protected function is_smtp_configured() {
+
+		if ( ! $this->is_smtp_activated() || ! class_exists( '\\WPMailSMTP\\Options' ) ) {
+			return false;
+		}
+
+		$phpmailer = $this->get_phpmailer();
+
+		$mailer             = \WPMailSMTP\Options::init()->get( 'mail', 'mailer' );
+		$is_mailer_complete = empty( $mailer ) ? false : wp_mail_smtp()->get_providers()->get_mailer( $mailer, $phpmailer )->is_mailer_complete();
```

Comment:
```suggestion
		$is_mailer_complete = ! empty( $mailer ) && wp_mail_smtp()->get_providers()->get_mailer( $mailer, $phpmailer )->is_mailer_complete();
```

Inconsequential, but we can simplify this a bit.
I